pom.xml

```diff
@@ -129,6 +129,12 @@
                 <version>3.0.1</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
```

Comment:
Fjern prosessering-jdbc fra modeules og slett pom.xml i prosessering-jdbc

Reply:
Jeg hadde først en tanke om at ikke fjerne den for at den då kunde fjernes først fra de som har brukt den, uten at man får feil som man kanskje ikke helt skjønner. 
Har fjernet den nå